File: Data Structures & Algorithms/sign-of-the-product-of-an-array/submission-1.java
class Solution {
    public int arraySign(int[] nums) {
        double product = 1;
        for (int num : nums) {
            product *= (double) num;
        }
        if (product > 0.0) {
            return 1;
        } else if (product < 0.0) {
            return -1;
        } else {
            return 0;
        }
    }
}

File: Data Structures & Algorithms/concatenation-of-array/submission-1.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        doubled = list(range(len(nums) * 2))
        for i in range(0, len(nums)):
            doubled[i] = nums[i]
            doubled[i + len(nums)] = nums[i]
        return doubled